### PR TITLE
Update 00-intro.md

### PR DESCRIPTION
**Pull request by finleyexp**
Add additional requirements

### CODE REVIEW

File: nameservice/tutorial/00-intro.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ All of the final source code for this tutorial project is in this directory (and
 
 - [`golang` >1.13.0](https://golang.org/doc/install) installed
 - A working [`$GOPATH`](https://github.com/golang/go/wiki/SettingGOPATH)
+- Github account and [Github Desktop (64-bit required)](https://help.github.com/en/desktop/getting-started-with-github-desktop/installing-github-desktop)
```
Comment (marbar3778) [inline nameservice/tutorial/00-intro.md]: Github desktop is not required to run this tutorial, nor github as you could download the source code from github if you wanted

Comment (finleyexp) [reply nameservice/tutorial/00-intro.md]: When closely following the tutorial, Git is the first instruction after setting up $GOPATH, which is a surprise for anyone who read the requirements. If you're not using the tutorial exactly as written, then no, not required.

Comment (marbar3778) [reply nameservice/tutorial/00-intro.md]: okay, we can leave GitHub account, but GitHub desktop Id advocate against as many people use the cli

Comment (finleyexp) [reply nameservice/tutorial/00-intro.md]: In that case, could we add the link to the [Github CLI](https://hub.github.com/) as well?

Comment (jackzampolin) [reply nameservice/tutorial/00-intro.md]: Well you just taught me a new one there @finleyexp, I thought the github cli was `git`.

Comment (finleyexp) [reply nameservice/tutorial/00-intro.md]: Wow, @jackzampolin, in your rush to sarcasm, it seems you missed that my last comment was a question about adding the link to <code>git</code>. I'm trying to help clarify the tutorial for users that may not know anything about Github. Does <code>git</code> install itself? No. How does a new user get <code>git</code>? The link I provided in the previous comment is one way.

**Comment by marbar3778** [comment]
Thank you for your contirbution 😄 <a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F143vPc6b08locw%2Fgiphy.gif&token=shipit" data-gitmeme-token="shipit"></a>

**Pull request merged**